release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -998,7 +998,7 @@ In the table, features are marked with the following statuses:
 |GA
 |DEP
 
-|Use of `v1` in `apiVersion` for {product-title} resources
+|Use of `v1` without a group in `apiVersion` for {product-title} resources
```

Comment:
@soltysh Can you let me know if this clarification is appropriate?

Preview: https://deploy-preview-31775--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-deprecated-removed-features

When I read it the first time it confused me into thinking it was all v1 references, even if there was a group in front of it.

Reply:
Yeah, this is good, thx!